packages/block-library/src/table/state.js

```diff
@@ -55,13 +55,9 @@ export function getFirstRow( state ) {
  */
 export function getCellAttribute( state, cellLocation, attributeName ) {
 	const { sectionName, rowIndex, columnIndex } = cellLocation;
-	return get( state, [
-		sectionName,
-		rowIndex,
-		'cells',
-		columnIndex,
-		attributeName,
-	] );
+	return state[ sectionName ]?.[ rowIndex ]?.cells?.[ columnIndex ]?.[
```

Comment:
`attributes` that populates the `state` property will always be an object, and the Table block already uses it as one. The rest may not exist, so we're adding some optional chaining.